test/e2e/routegroup.go

```diff
@@ -390,6 +390,8 @@ rBackend: Path("/backend") -> inlineContent("%s") -> <shunt>;
 			return code == http.StatusTooManyRequests
 		}, false)
 		Expect(err).NotTo(HaveOccurred())
+		s, err = getBody(resp)
```

Comment:
this is wrong, because test is:
```
By("checking the response is for a request to /backend with the right header we know if we got the correct route but get ratelimited")
```
and the response is:
```
resp, err = waitForResponseReturnResponse(req, 5*time.Minute, func(code int) bool {
			return code == http.StatusTooManyRequests
		}, false)
```

so we wait for status code 429 and then you compare the body with the initial expected body if we are not rate limited.

Reply:
Oh, so you mean there's no need to check response body, and we can just check if the returned code is 429, that implies rate limiting?

Reply:
I added a commit to only check status code, testing local takes forever so I pushed it.

Reply:
still failing 😅 but not because of this test case I think. Or I need to restart the full pipeline not only e2e step.

Reply:
Thanks! I retriggered the pipeline 😅

Reply:
It passed now 🙂